cpp/inc/bond/ext/grpc/detail/client_call_data.h

```diff
@@ -57,7 +57,7 @@ struct client_unary_call_data : io_manager_tag
         std::shared_ptr<grpc::ChannelInterface> channel,
         std::shared_ptr<io_manager> ioManager,
         std::shared_ptr<TThreadPool> threadPool,
-        CallbackType cb)
+        CallbackType cb = CallbackType())
```

Comment:
I think it is [aggregate_initialization](http://en.cppreference.com/w/cpp/language/aggregate_initialization).